src/node_crypto.cc

```diff
@@ -1154,6 +1163,12 @@ void SSLWrap<Base>::AddMethods(Environment* env, Handle<FunctionTemplate> t) {
   env->SetProtoMethod(t, "setNPNProtocols", SetNPNProtocols);
 #endif
 
+#ifdef TLSEXT_TYPE_application_layer_protocol_negotiation
+  NODE_SET_PROTOTYPE_METHOD(t, "getALPNNegotiatedProtocol",
```

Comment:
I removed #ifdef here and fixed to have declaration to avoid compile errors and also changed `NODE_SET_PROTOTYPE_METHOD` macro into ` env->SetProtoMethod`.